js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,125 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+const siblingsToggle = '[data-next-siblings-toggle]';
+const siblingsToggleInputs = `${siblingsToggle} input[type="radio"]`;
+
+function getResponseReason(resp) {
```

Comment:
Okay.

The property getting idiom using `&&` to test the chain or falling back to a default is an idiom that is widely used. 

My feeling is this type of code is not really clear and direct as to its purpose. One reason is because its purpose is to do this:

```js
const deepValue = this.is.a.long.property.path;
if (!deepValue) deepValue = defaultValue;
```

Which is clear and direct about what is intended. 

The one line version using Boolean short-circuit evaluation is an idiom that probably is clear to a person if the person already knows the idiom, or if a person works out what the evaluation would be.